docs/dev_guide/diagnostic_ref/dpct1045.rst

```diff
@@ -100,12 +102,15 @@ which is rewritten to:
                              std::shared_ptr<dpct::sparse::matrix_info> descrA,

```

Comment:
In line 102,  let's add a new pseudo function:  convert_data_fomat()

Reply:
done